packages/apollo-codegen-swift/src/language.ts

```diff
@@ -311,12 +311,33 @@ export class SwiftGenerator<Context> extends CodeGenerator<
     super(context);
   }
 
-  multilineString(string: string) {
+  /**
+   * Outputs a multi-line string
+   *
+   * @param string - The Multi-lined string to output
+   * @param suppressMultilineStringLiterals - If true, will output the multiline string as a single trimmed
+   *                                          string to save bandwidth.
+   *                                          NOTE: This preference will be ignored if your GraphQL query
+   *                                          contains a triple-quote GraphQL string literal, since that's
+   *                                          the same as Swift's string literal and will break the query.
+   */
+  multilineString(string: string, suppressMultilineStringLiterals: Boolean) {
     // Disable trimming if the string contains """ as this means we're probably printing an
     // operation definition where trimming is destructive.
-    this.printOnNewline(
-      SwiftSource.string(string, /* trim */ !string.includes('"""'))
-    );
+    if (string.includes('"""')) {
+      this.printOnNewline(SwiftSource.string(string, /* trim */ false));
```

Comment:
We don't have to do this. We can instead tweak escaping on the multiline string form such that any sequence of three quotes in a row has a backslash before the first quote. The following is perfectly legal and represents 9 quotes in a row:

```
"""
\"""\"""\"""
"""
```

Reply:
This sounds like a great idea for a follow-up PR.